tests/cypress/integration/pro/subscriptions_spec.js

```diff
@@ -1,5 +1,7 @@
 /// <reference types="cypress" />
 
+Cypress.config("defaultCommandTimeout", 10000);
```

Comment:
[Cypress.config() changes the configuration for the remaining execution of the current spec file](https://docs.cypress.io/guides/references/configuration#Cypressconfig), but it's not immediately obvious (I actually had to refer to the documentation to double check this).

For that reason I'd recommend to avoid it. If you need to customise the timeout for a specific test/group of tests, it might be better to pass it as a configuration option, as explained in the docs. It is more self-explainatory as it's tied to a particular function.
 https://docs.cypress.io/guides/references/configuration#Syntax

Reply:
From the docs I see that there is nothing wrong with using this statement. And since it's used in all other test files in the project, it's consistent. We can add a comment on top of this statement to make it more clear what it does. Wdyt?